### PR TITLE
Alignment is always a set of tuples

### PR DESCRIPTION
**Pull request by natsegal**
PN9 interface will be updated accordingly

### CODE REVIEW

File: nmtwizard/preprocess/tu.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def alignment(self):
         if self.__alignment is None:
             return None
         self._initialize_alignment()
-        return [set(part) for part in self.__alignment.alignments]
+        return [part for part in self.__alignment.alignments]
```
Comment (guillaumekln) [inline nmtwizard/preprocess/tu.py:253]: Shouldn't we return a copy here?

File: nmtwizard/preprocess/tu.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def align(self, src_tok, tgt_tok):
             for src_tok_part, tgt_tok_part in zip(src_tok, tgt_tok):
                 align_result = self.aligner.align(src_tok_part, tgt_tok_part)
                 # TODO : write fwd and bwd probs
-                alignments.append(set(align_result["alignments"]))
+                alignments.append(align_result["alignments"])
```
Comment (guillaumekln) [inline nmtwizard/preprocess/tu.py:73]: I think the module `systran_align` will keep returning a list. So we should probably keep the conversion.

File: nmtwizard/preprocess/tu.py (hunk 1 above)
Comment (natsegal) [reply nmtwizard/preprocess/tu.py:253]: Yes, you're right.

File: nmtwizard/preprocess/tu.py (hunk 2 above)
Comment (natsegal) [reply nmtwizard/preprocess/tu.py:73]: Yes, you are right again. Sorry, I'm starting to loose track a bit.

**Review by guillaumekln — APPROVED** [review]

**Pull request merged**